packages/core-data/src/actions.js

```diff
@@ -153,6 +153,25 @@ export function __experimentalReceiveThemeBaseGlobalStyles(
 	};
 }
 
+/**
+ * Returns an action object used in signalling that the theme global styles variations have been received.
+ *
+ * @param {string} stylesheet The theme's identifier
+ * @param {Array}  variations The global styles variations.
+ *
+ * @return {Object} Action object.
+ */
+export function __experimentalReceiveThemeGlobalStyleVariations(
```

Comment:
Can't we use the entities mechanism to add an entity in packages/core-data/src/entities.js for style variations and avoid the need to create an action a reducer etc?

Reply:
I'm not sure it's possible because it depends on the "theme" but that's a good remark, we can think about it more.